src/common/wui_api.c

```diff
@@ -247,7 +247,7 @@ void sntp_get_system_date(char * dest){
 
     HAL_RTC_GetDate(&hrtc, &currDate, RTC_FORMAT_BIN);
 
-    snprintf(dest, 13, "%02d.%02d.%d", currDate.Date, currDate.Month + 1, currDate.Year + 1900);
+    snprintf(dest, 13, "%d.%d.%d", currDate.Date, currDate.Month + 1, currDate.Year + 1900);
   } else {
```

Comment:
This change is unnecessary. There was some misunderstanding about what is problem with formatting. Even the previous commit build successfully without warnings. You can remove this commit if you don't like it.

Reply:
commit `1a0e835`